bundles/org.openhab.binding.androidtv/pom.xml

```diff
@@ -33,6 +37,36 @@
       <version>1.75</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient-osgi</artifactId>
+      <version>4.5.14</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpcore-osgi</artifactId>
+      <version>4.4.16</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
+      <version>${jackson.version}</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
+      <version>${jackson.version}</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>${jackson.version}</version>
+      <scope>compile</scope>
+    </dependency>
   </dependencies>
```

Comment:
Why are these dependency's introduced? They did not seem to be essential for the other Things in this binding. 
If they cannot be removed, they should also be mentioned in the notice file.

Reply:
I'm not a maven guy, not entirely sure what you're saying here.  The Philips protocol requires the jackson xml bits and apache http.  I'm not sure how to only set the dependency on a per thing type basis.

Reply:
Sounds good.  I'm about to start working on this so you should see some commits in a bit.

Reply:
@jlaur looking at https://github.com/openhab/openhab-addons/pull/16311 i was comparing some dependency files.  I noticed that these `httpclient` dependencies in the boschindigo binding are also listed in the feature.xml file. 
That seemed somewhat unexpected as these are in the pom.xml set to `compile`, am i missing something? Should these bundles also be added to the feature.xml in this binding?